### ISSUE TITLE
Probe customer interest in a Resource feature

### ISSUE THREAD

**Issue by pyrello** [uiowa/uiowa#6723]
# Motivation

People have expressed interest in some type of generic resource content type that can either represent information that is internal or external to a site. Moreover, it has been suggested that there might be some overlap between this feature and building a service content type for ITS. We would like to better understand what our customers want in a resource feature so that we can decide whether to build the ITS services content type as an extension of that feature.

# Proposed solution
- Put together a list of customers who have previously expressed interest in this type of feature
- Get in touch with those customers to find out what things are important to them about how it would work
- Compare that to what we know about the ITS service content type
- Provide a recommendation to the team about whether we would base the ITS service content type on this feature or not

**Comment by e-marie-w** [comment]
Identify list of customers who have expressed interest and invite them to share their thoughts at demo.

**Comment by bspeare** [comment]
We had a previous discussion in June on how we might handle this with the page content type and a specialized Resource content type: https://github.com/uiowa/uiowa/issues/6420#issuecomment-1591995158.

**Comment by cory-skeers** [comment]
From Patrick R in demo discussion:

Categorizations through multiple taxonomies? Eg, can have a taxonomy for degree type and one for department, and then tag individual pages as "bachelor's, social work" and then have a page displaying two filters, one for each.

2 examples of requests he's received:
1. Catalog of databases that contain certain types of information, and would like a filterable guide to these databases. "I want to see all databases that have info on smoking cessation, and choose that from a dropdown to see this collection"
2. Collections of people who work within their lab

Basically, looking for filterable lists (or faceted search).

Interested in being able to tag content, but specifically to be able to build a customized search panel to filter these.

**Comment by cory-skeers** [comment]
"i don't really have any preference for a new content type. it's more that the page content type could have all the metadate that articles have, and the ability to be sorted and filtered in lists like news. or give the article type the layout power of the page content type"

**Comment by cory-skeers** [comment]
"Personally, I love the basic page.  I would love to have a way to apply an attribute on a basic page that says, “This is part of Group X” (where X is a taxonomy term).

Once I’ve specific that, the corresponding page becomes discoverable in an internal search and dynamic filtering solution for X like we see on Program Finder/Areas of Study Finder...

Search would include meta, title, page content, taxonomy, and perhaps summary field.

Summary field would be displayed in the search machine.

Other taxonomy terms inside a facet would provide a filterable experience (if used) like we see on Classrooms.

I don't want to have to ask for a custom content type to do this type of thing.  I want to use basic pages for my content, have taxonomy drive the experience, etc

I’ll also note that I quite enjoy the dynamic display we see on Classrooms (and elsewhere).  

I am sure there are other use cases where it makes sense to define a custom content type and all the attributes, etc…  I’m just saying I kind of want the opposite.  I want the faceted browse and search functionality applied to subsets of basic pages that I define…

As a product sponsor on the team, I don’t want to have to always build a custom content type for anything that wants to use this type of functionality. 

Oh, and to throw more complexity in the mix, as a user I don’t want the solution to be opinionated regarding whether or not the pages that are included are basic pages or news articles…. 

As a user, I would be OK with a solution that required a Megan or a Michael to help with setting up (provided they had capacity to work with us in a timely manner).  As a Michael, I might be a bit worried about capacity"

**Comment by joewhitsitt** [comment]
>For faceted browsing experience, I’d like options to display results (dynamically) as list/table/or Card.

**Comment by mjoneill** [comment]
More data coming in:

> The site will include includes pages introducing hundreds of well-being resources. We seek a multifaceted search/filter tool that lets users narrow lists of resources by selecting tags for topic, audience, type, and potentially other categories. Similar search tools exist on UI sites like [admissions](https://admissions.uiowa.edu/areas-of-study) and [College of Education](https://education.uiowa.edu/areas-study/program-finder). The well-being site project team has identified taxonomy categories and potential tags, but deploying a tool that uses them will require custom development (or adapting one of the existing UI models).

Source: Wellness Group

**Comment by cory-skeers** [comment]
Might want to consider health topics build like https://uihc.org/health-topics as another reference point for a similar type of functionality. The Wellness Group request reminded me of it.

**Comment by pyrello** [comment]
> Add a way for search results to be filtered to a particular set of content
Requested by Scott Leeman for https://epicsupport.sites.uiowa.edu/epic-courses

**Comment by joewhitsitt** [comment]
Found this similar issue in the backlog https://github.com/uiowa/uiowa/issues/4229
